src/libsbn.cpp

```diff
@@ -85,6 +87,19 @@ void SBNInstance::TrainExpectationMaximization(double alpha, size_t em_loop_coun
       parent_to_range_, alpha, em_loop_count);
 }
 
+EigenVectorXd SBNInstance::CalculateSBNProbabilitiesInLog() {
+  // invariant: sbn_parameters_ are  in the log space i.e.,
```

Comment:
I don't honestly know what "invariant" means in this context!

Reply:
I meant [class invariant](https://en.wikipedia.org/wiki/Class_invariant). Basically writing out our expectation that the sbn_parameters_ to always be in (unnormalized) log space. I removed this comment.